app/models/concerns/movements.rb

```diff
@@ -0,0 +1,21 @@
+module Movements
+
+  def starting_point_x
+    @starting_point_x = self.x_pos
+  end
+
+  def starting_point_y
+    @starting_point_y = self.y_pos
+  end
+
+  def right_or_left(destination_x)
+    return 1 if starting_point_x < destination_x

```

Comment:
Lines 12, 13, 17, and 18 can be refactored using ternary operator and be a one liner code